base/build.gradle

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+import org.apache.tools.ant.taskdefs.condition.Os
+
+description = 'Ballerinax - Docker Base'
+
+configurations {
+    bDistributionDir
+}
+
+jar {
+    enabled = false
+}
+
+dependencies {
+    bDistributionDir project(path: ":docker-extension-annotations", configuration: "bDistributionDir")
+}
+
+def distributionLocation = project(":docker-extension-annotations").buildDir.absolutePath + "/extracted-distribution/jballerina-tools-${ballerinaLangVersion}"
+
+task copyRuntimeJar {
+    dependsOn configurations.bDistributionDir
+    doLast {
+        copy {
+            from file("$distributionLocation/bre/lib/ballerina-rt-${ballerinaLangVersion}.jar")
```

Comment:
We can use like following instead of deriving from `project(":docker-extension-annotations").buildDir.absolutePath`.

"$configurations.bDistributionDir.singleFile.absolutePath"